Examples/Cosmo/params.txt

```diff
@@ -0,0 +1,184 @@
+# See the wiki page for an explanation of the params!
+# https://github.com/GRChombo/GRChombo/wiki/Guide-to-parameters
+
+#################################################
+# Filesystem parameters
+
+verbosity = 0
+
+# location / naming of output files
+# output_path = "" # Main path for all files. Must exist!
+chk_prefix = Cosmo_
+plot_prefix = Cosmop_
+# restart_file = Cosmo_000000.3d.hdf5
+
+# HDF5files are written every dt = L/N*dt_multiplier*checkpoint_interval
+checkpoint_interval = 1000
+# set to 0 to turn off plot files (except at t=0 and t=stop_time)
+# set to -1 to never ever print plotfiles
+plot_interval = 500
+num_plot_vars = 7
+plot_vars = chi phi rho_scaled S_scaled K_scaled Ham Mom
+
+# subpaths - specific directories for hdf5, pout, extraction data
+# (these are created at runtime)
+hdf5_subpath = "hdf5"
+pout_subpath = "pout"
+data_subpath = "data"
+
+# change the name of output files
+# pout_prefix = "pout"
+print_progress_only_to_rank_0 = 1
+
+# ignore_checkpoint_name_mismatch = 0
+# write_plot_ghosts = 0
+
+#################################################
+# Initial Data parameters
+
+# Change the gravitational constant of the Universe!
+# Default is 1.0, for standard geometric units
+# Here we decouple the evolution so the scalar evolved on the
+# metric background without backreaction (this avoids the need
+# to solve the constaints)
+G_Newton = 1.
+
+# Scalar field initial data
+scalar_amplitude = 0.0777635
+scalar_width = 1.0 # not use
+scalar_mass = 1.0
+scalar_field_mode = 1.0
+
+# lineout params
+lineout_num_points = 10
```

Comment:
Add lineout_radius? (if defaulted can comment out, but still list it in params.)